module/skill/QuinticWalk/src/QuinticWalk.cpp

```diff
@@ -0,0 +1,383 @@
+#include "QuinticWalk.hpp"
+
+#include <fmt/format.h>
+
+#include "extension/Configuration.hpp"
+
+#include "message/actuation/KinematicsModel.hpp"
+#include "message/actuation/Limbs.hpp"
+#include "message/actuation/LimbsIK.hpp"
+#include "message/actuation/ServoCommand.hpp"
+#include "message/behaviour/Behaviour.hpp"
+#include "message/behaviour/state/Stability.hpp"
+#include "message/motion/GetupCommand.hpp"
+#include "message/skill/Walk.hpp"
+#include "message/support/nusight/DataPoint.hpp"
+
+#include "utility/actuation/InverseKinematics.hpp"
+#include "utility/math/comparison.hpp"
+#include "utility/math/euler.hpp"
+#include "utility/nusight/NUhelpers.hpp"
+#include "utility/support/yaml_expression.hpp"
+
+namespace module::skill {
+
+    using extension::Configuration;
+
+    using message::actuation::KinematicsModel;
+    using message::actuation::LeftLegIK;
+    using message::actuation::RightLegIK;
+    using message::actuation::ServoCommand;
+    using message::behaviour::Behaviour;
+    using message::behaviour::state::Stability;
+    using message::input::Sensors;
+    using message::skill::Walk;
+
```

Comment:
I'm not sure formatting has applied properly it's supposed to sort all these usings